### DISCUSSION TITLE
about pyaedt cutout （hfss3Dlayout）

### DISCUSSION THREAD

**Discussion by Roy-debug-hub** [ansys/pyaedt#5391] [Q&A] [answered]
How can I use pyaedt to cutout a pcb layout and generate a new design in the project?

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @Roy-debug-hub , the cutout can be performed directly from PyEDB. Please check this example:

https://examples.aedt.docs.pyansys.com/version/dev/examples/high_frequency/layout/power_integrity/ac_q3d.html

**Reply by Roy-debug-hub** [reply]
Thanks for your reply, now I can cut and export a design from a project as an aedb file, but how can I add my cutout aedb file as a second design under the original project ?  note: here is the main code I use 
`from ansys.aedt.core import Hfss3dLayout
ADET_VERSION = 2023.2
NUM_CORE = 4
NG_MODE = Flase
origin_pcb_path = './PCB.aedb'

h3d = Hfss3dLayout(
project = origin_pcb_path ,
version = AEDT_VERSION,
non_graphical = NG_MODE,
new_desktop = True)

polygon = [
[-4000, 400],
[-500, 300],
[-500, 4500],
[-4000, 4500]
]

h3d.modeler.edb.cutout(
custom_extent = polygon,
custom_extent_units='um',
output_aedb_path = './pcb_cut.aedb',
open_cutout_at_end = True,
preserve_components_with_model = True
)`
